util/embedded_target.py

```diff
@@ -29,13 +29,16 @@ def run_objdump(objdump, input, basename, outdir):
     subprocess.run(cmd, stdout=f, check=True)
     return output
 
+
```

Comment:
ALl changes to this file were made by `lintpy.py`.

Reply:
Please remove them from this PR by selectively adding only your changes. `yapf` doesn't produce consistent output unfortunately.